samples/azure/azure-service-fabric-routing/sample.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Azure Service Fabric Routing
+reviewed: 2016-10-10
+component: ASF
```

Comment:
do u need help adding ASF to https://github.com/Particular/docs.particular.net/blob/master/components/components.yaml  ?